src/material/schematics/ng-generate/nav/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.ts.template

```diff
@@ -20,6 +20,7 @@ export class <%= classify(name) %>Component {
 
   isHandset$: Observable<boolean> = this.breakpointObserver.observe(Breakpoints.Handset)
     .pipe(
+      share(),
```

Comment:
Isn't the `share` supposed to be at the end after the `map`?

Reply:
Made the change - however looking at the `share()` operator it seems like the order doesn't matter as it will still emit only once